graylog-storage-opensearch2/src/main/java/org/graylog/storage/opensearch2/stats/ClusterStatsApi.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.storage.opensearch2.stats;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.inject.Inject;
+import org.graylog.storage.opensearch2.PlainJsonApi;
+import org.graylog2.rest.resources.system.indexer.responses.IndexSetStats;
+import org.opensearch.client.Request;
```

Comment:
This is still using the "opensearch-java" client. The client is not available anymore because we reverted it. So you have to port it over to the OpenSeach high-level REST client.

Reply:
fixed